src/runtime_src/core/tools/common/tests/TestAIEReconfigOverhead.cpp

```diff
@@ -0,0 +1,163 @@
+  xrt::kernel kernel;
+  try {
+    hwctx = xrt::hw_context(working_dev, xclbin.get_uuid());
+    kernel = xrt::kernel(hwctx, kernelName);
+  } 
+  catch (const std::exception& ex)
+  {
+    logger(ptree, "Error", ex.what());
+    ptree.put("status", test_token_failed);
+    return ptree;
+  }
+
+  const auto seq_name = xrt_core::device_query<xrt_core::query::sequence_name>(dev, xrt_core::query::sequence_name::type::aie_reconfig_overhead);
+  logger(ptree, "Debug", boost::str(boost::format("DPU sequence path: '%s'") % seq_name));
+  auto dpu_instr = findPlatformFile(seq_name, ptree);
+  if (!std::filesystem::exists(dpu_instr))
+    return ptree;
+
+  logger(ptree, "DPU-Sequence", dpu_instr);
+
+  size_t instr_size = 0;
+  try {
+    instr_size = get_instr_size(dpu_instr); 
+  }
+  catch(const std::exception& ex) {
+    logger(ptree, "Error", ex.what());
+    ptree.put("status", test_token_failed);
+    return ptree;
+  }
+
+  //Create BOs
+  xrt::bo bo_ifm(working_dev, buffer_size, XRT_BO_FLAGS_HOST_ONLY, kernel.group_id(1));
+  xrt::bo bo_ofm(working_dev, buffer_size, XRT_BO_FLAGS_HOST_ONLY, kernel.group_id(3));
+  xrt::bo bo_instr(working_dev, instr_size*sizeof(int), XCL_BO_FLAGS_CACHEABLE, kernel.group_id(5));
+
+  init_instr_buf(bo_instr, dpu_instr);
+
+  // map input buffer
+  auto ifm_mapped = bo_ifm.map<int*>();
+  for (size_t i = 0; i < word_count; i++)
+    ifm_mapped[i] = rand() % 4096;
+
+  //Sync BOs
+  bo_instr.sync(XCL_BO_SYNC_BO_TO_DEVICE);
+  bo_ifm.sync(XCL_BO_SYNC_BO_TO_DEVICE);
+
+  //Log
+  if(XBUtilities::getVerbose()) { 
+    logger(ptree, "Details", boost::str(boost::format("Buffer size: '%f'GB") % buffer_size_gb));
+    logger(ptree, "Details", boost::str(boost::format("No. of iterations: '%f'") % itr_count));
+  }
+
+  auto start = std::chrono::high_resolution_clock::now();
+  try{
+    auto run = kernel(1, bo_ifm, NULL, bo_ofm, NULL, bo_instr, instr_size, NULL);
+    run.wait2();
+  }
+  catch (const std::exception& ex)
+  {
+    logger(ptree, "Error", ex.what());
+    ptree.put("status", test_token_failed);
+    return ptree;
+  }
+  auto end = std::chrono::high_resolution_clock::now();
+  float elapsedSecs = std::chrono::duration_cast<std::chrono::duration<float>>(end-start).count();
+
+  start = std::chrono::high_resolution_clock::now();
+  for (int i = 0; i< itr_count; i++)
+  {
+    try{
+      auto run = kernel(1, bo_ifm, NULL, bo_ofm, NULL, bo_instr, instr_size, NULL);
+      run.wait2();
+    }
+    catch (const std::exception& ex)
+    {
+      logger(ptree, "Error", ex.what());
+      ptree.put("status", test_token_failed);
+      return ptree;
+    }
+  }
+  end = std::chrono::high_resolution_clock::now();
+  float elapsedSecsAverage = std::chrono::duration_cast<std::chrono::duration<float>>(end-start).count();
+  elapsedSecsAverage /= itr_count;
+  if(XBUtilities::getVerbose()){
+    logger(ptree, "Details", boost::str(boost::format("ElapsedSec: '%.1f' us") % (elapsedSecs * 1000000)));
```

Comment:
Let's print this is seconds for consistency and instead of "ElapsedSec" let's name it "Total Duration"